tests/e2e/local/minikube/setup_host.sh

```diff
@@ -14,12 +14,13 @@ case "${OSTYPE}" in
   *) echo "unsupported: ${OSTYPE}" ;;
 esac
 
-# shellcheck disable=SC2153
-if [ ! -z "$VM_DRIVER" ]; then
-  vm_driver="$VM_DRIVER"
-fi
+# VM for Minikube, the default is decided by your os type if not specified
```

Comment:
What about change the comment to:

Virtual machine driver for minikube, the default value is decided by your OS type if it's not specified...